Build failure for Example Generators

Compilation error, due to the fact **webdevice.jar** is removed from examples generators
```
[ERROR] Failed to execute goal on project generator-runner: Could not resolve dependencies for project org.eclipse.vorto.examples:generator-runner:jar:0.0.1-SNAPSHOT: Could not find artifact org.eclipse.vorto.examples:org.eclipse.vorto.codegen.webdevice:jar:0.0.1-SNAPSHOT -> [Help 1]
[ERROR] 
```

--- a/vorto-generators/generator-runner/pom.xml
+++ b/vorto-generators/generator-runner/pom.xml
@@ -106,11 +106,6 @@
 		</dependency>
 		<dependency>
 			<groupId>org.eclipse.vorto.examples</groupId>
-			<artifactId>org.eclipse.vorto.codegen.webdevice</artifactId>
-			<version>${project.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.vorto.examples</groupId>
 			<artifactId>org.eclipse.vorto.codegen.webui</artifactId>
 			<version>${project.version}</version>
 		</dependency>

--- a/vorto-generators/generator-runner/src/main/java/org/eclipse/vorto/examples/generators/GeneratorConfiguration.java
+++ b/vorto-generators/generator-runner/src/main/java/org/eclipse/vorto/examples/generators/GeneratorConfiguration.java
@@ -27,7 +27,6 @@ import org.eclipse.vorto.codegen.protobuf.ProtobufGenerator;
 import org.eclipse.vorto.codegen.spi.config.AbstractGeneratorConfiguration;
 import org.eclipse.vorto.codegen.spi.model.Generator;
 import org.eclipse.vorto.codegen.thingworx.ThingWorxCodeGenerator;
-import org.eclipse.vorto.codegen.webdevice.WebDeviceGenerator;
 import org.eclipse.vorto.codegen.webui.WebUIGenerator;
 import org.springframework.stereotype.Component;
 
@@ -47,7 +46,6 @@ public class GeneratorConfiguration extends AbstractGeneratorConfiguration {
 		addGenerator(Generator.create("/generators/markdown.properties", MarkdownGenerator.class));
 		addGenerator(Generator.create("/generators/protobuf.properties", ProtobufGenerator.class));
 		addGenerator(Generator.create("/generators/thingworx.properties", ThingWorxCodeGenerator.class));
-		addGenerator(Generator.create("/generators/webdevice.properties", WebDeviceGenerator.class));
 		addGenerator(Generator.create("/generators/webui.properties", WebUIGenerator.class));
 	}
 	

